1.Exploratory_Data_Analysis/scripts/2b.correlation_agg_morphology_features.py

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Correlation of Aggregated Features
+# Each well is median aggregated and normalized.
+# The correlation of the aggregated features is calculated per:
+# * well
+# * per treatment
+# * per selected treatment
+
+# In[1]:
+
+
+import pathlib
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import plotnine as gg
+import scipy.stats as stats
+import seaborn as sns
+import statsmodels.api as sm
+
+# In[2]:
+
+
+# Parameters
+cell_type = "PBMC"
+
+
+# In[3]:
+
+
+# set import data paths
+feature_df_path = pathlib.Path(f"../data/{cell_type}_sc_aggregate_norm.parquet")
+
+# read in data
+feature_df = pd.read_parquet(feature_df_path)
+
+
+# ## Clean up df
+
+# In[4]:
+
+
+# remove uM in each row of the Metadata_inducer1_concentration column if it is present
```

Comment:
maybe i'm not reading closely enough, but it seems there is a lot of repeated code here. If you refactor to clean the data first, then you won't need to clean it every time.

Reply:
added '0.nomic_cleanup.ipynb` to adress this comment